src/NLog/Targets/TargetWithContext.cs

```diff
@@ -516,7 +509,7 @@ protected virtual bool SerializeMdcItem(LogEventInfo logEvent, string name, obje
             foreach (var name in names)
             {
                 object value = MappedDiagnosticsLogicalContext.GetObject(name);
-                if (SerializeMdlcItem(logEvent, name, value, out var serializedValue))
+                if (SerializeMdlcItem(logEvent, name, value, out _))
```

Comment:
@snakefoot is this correct we aren't using the serializedValue? (2 times)

Reply:
No that is a bug.

Reply:
OK fixing this now